docs/examples/util/check_predictions.py

```diff
@@ -0,0 +1,84 @@
+import os
```

Comment:
A script to inspect the prediction results of the 3 examples:
- docs/algo/complete-example.py
- docs/opf/complete-example.py
- docs/networkapi/complete-example.py

Adding this info at the top of the file.